src/polyfills.js

```diff
@@ -0,0 +1,2 @@
+// Entrypoint for browser polyfills
```

Comment:
comment... add references to find polyfills (polyfill.io, mdn, etc.)

Reply:
Opting to not add comment. I updated the [wiki](https://github.com/rackerlabs/helix-ui/wiki/Compatibility#polyfills) to include more information about polyfills, given that the information is relevant to HelixUI developers, not the consumers.